File: demo-server/src/main/java/com/example/demoserver/model/Article.java
package com.example.demoserver.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Article {

    @Id
    private Long id;
    private String title;
    private String content;
    private String userId;
}

File: demo-front/src/model/Article.js
class Article {
    constructor(id, title, content, userId) {
      this.title = title;
      this.content = content;
      this.userId = userId;
      this.id = id;
    }
  
    getTitle() {
      return this.title
    }
  
    getContent() {
      return this.content
    }
  
    getUserId() {
      return this.userId
    }
  
    getId() {
      return this.id
    }
  }
    
  export default Article;